### PR TITLE
Feat/Small video component

### PR DESCRIPTION
**Pull request by micblyth**
- Renamed suggested-videos to small-video-previews
- made small-video-previews its own component

### CODE REVIEW

File: src/ui/VideoPreview/index.tsx [hunk 1]
```diff
@@ -1,9 +1,9 @@
 import { ReactElement } from 'react'
-import VideoDetails from './VideoDetails'
+import LargeVideoDetails from './LargeVideoDetails'
 import './styles.css'
 
 interface VideoPreviewProps {
-  image: string
+  largevideoimage: string
```
Comment (jarkin13) [inline src/ui/VideoPreview/index.tsx]: camel case and why large video when component is `VideoPreview`?

File: src/ui/SmallVideoPreview/index.tsx [hunk 2]
```diff
@@ -0,0 +1,38 @@
+import { ReactElement } from 'react'
+import './styles.css'
+import SmallVideoDetails from '../SmallVideoPreview/SmallVideoDetails'
+
+interface VideoPreviewProps {
+  smallvideoimage: string
```
Comment (jarkin13) [inline src/ui/SmallVideoPreview/index.tsx]: camel case

File: src/ui/SmallVideoPreview/index.tsx [hunk 3]
```diff
@@ -0,0 +1,38 @@
+import { ReactElement } from 'react'
+import './styles.css'
+import SmallVideoDetails from '../SmallVideoPreview/SmallVideoDetails'
+
+interface VideoPreviewProps {
```
Comment (jarkin13) [inline src/ui/SmallVideoPreview/index.tsx]: Props name should match component name
```suggestion
interface SmallVideoPreviewProps {
```

File: src/ui/SmallVideoPreview/SmallVideoDetails.tsx [hunk 4]
```diff
@@ -0,0 +1,31 @@
+import { ReactElement } from 'react'
+
+interface VideoDetailsProps {
```
Comment (jarkin13) [inline src/ui/SmallVideoPreview/SmallVideoDetails.tsx]: Props should match component name

File: src/ui/VideoPreview/LargeVideoDetails.tsx [hunk 5]
```diff
@@ -1,31 +1,31 @@
...
+  largeVideoTitle: string
   author: string
   verified: boolean
   numberOfViews: string
   timePostedAgo: string
 }
 
-const VideoDetails = ({
-  title,
+const LargeVideoDetails = ({
```
Comment (jarkin13) [inline src/ui/VideoPreview/LargeVideoDetails.tsx]: Also the component name(based on the folder) is `VideoPreview` why is this labeled as large video?

**Review by jarkin13 — CHANGES_REQUESTED** [review]

**Comment by jarkin13** [comment]
@micblyth  a few things:
- make sure prop names match the component's name
- make sure props are `camelCase` read more about that here https://www.w3schools.com/js/js_conventions.asp
- either change `VideoPreview` to `LargeVideoPreview` or keep as `VideoPreview`and make sure prop names reflect this
- the like icon is placed randomly in `/src` this should be moved to an image folder or where ever all the icons live

File: src/ui/VideoPreview/index.tsx (hunk 1 above)
Comment (micblyth) [reply src/ui/VideoPreview/index.tsx]: I wanted to changed everything over to Large Video for clear specification. Forgot to change everything over in a few spots.

File: src/ui/SmallVideoPreview/SmallVideoDetails.tsx [hunk 6]
```diff
@@ -0,0 +1,31 @@
+import { ReactElement } from 'react'
+
+interface SmallVideoProps {
```
Comment (jarkin13) [inline src/ui/SmallVideoPreview/SmallVideoDetails.tsx]:
```suggestion
interface SmallVideoDetailsProps {
```

File: src/ui/LargeVideoPreview/LargeVideoDetails.tsx [hunk 7]
```diff
@@ -1,31 +1,31 @@
 import { ReactElement } from 'react'
 
 interface VideoDetailsProps {
```
Comment (jarkin13) [inline src/ui/LargeVideoPreview/LargeVideoDetails.tsx]:
```suggestion
interface LargeVideoDetailsProps {
```

**Review by jarkin13 — CHANGES_REQUESTED** [review]

**Review by jarkin13 — APPROVED** [review]

**Pull request merged**